Docs/Configuration.tex

```diff
@@ -2986,9 +3006,9 @@ \subsection{Boot Properties}\label{miscbootprops}
   \texttt{HideAuxiliary}\\
   \textbf{Type}: \texttt{plist\ boolean}\\
   \textbf{Failsafe}: \texttt{false}\\
-  \textbf{Description}: Hides auxiliary entries from picker menu by default.
+  \textbf{Description}: Hides auxiliary entries from the picker menu unless overriden.
```

Comment:
Some verbs of `Description` part have imperative forms. It should always be consistent I guess. So, change `Hides` to `Hide`.

Reply:
I do not understand what "unless overridden" (two d actually) refers to. You cannot override that in the plist. The only way to see them is by entering the Advanced mode.

Reply:
> change Hides to Hide

The description refers to what the subject, HideAuxiliary, does. It hides xyz.

Reply:
>  do not understand what "unless overridden" (two d actually) refers to

Removed. Aim is met in the notes